experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -181,26 +179,31 @@ func (r *ExpanderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		// ------------------- EXPANSION SECTION -----------------------
 		logger = loggerCR.WithName(expander.Name).WithName("Expand")
 
-		expanderVersion, expanderRegistry, err := r.getExpanderVersion(ctx, expander.Version, expander.Type)
+		version := expander.Version
+		if version == "" {
```

Comment:
The version will default to latest (reference https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/experiments/compositions/composition/api/v1alpha1/composition_types.go#L74) so it will not be an empty string even when not specified.

Reply:
Thats a good point.
This is a defensive check. 

Lets have it.

Reply:
I have fixed this in a subsequent PR.

Reply:
fixed in an update to this pr